gf-webapp-admin/src/component/rooms/room/promo/Promo.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import styles from "./promo.module.scss";
+
+export const Promo = ({room}:{room:IRoom}) => {
+  const [isHover, setIsHover] = useState(false);
+  const [imgChangeModal, setImgChangeModal] = useState(false);
+  const [promoUrl, setPromoUrl] = useState("");
```

Comment:
What is promo URL? What is differnse to another images in slider?

Reply:
Naming has been changed